src/carma-1-tenth/run_c1t_metric_analysis.py

```diff
@@ -100,15 +90,20 @@ def test_C1T_07_slowdown_on_turns(self):
         speeds_on_turns = velocities[relevant_velocity_idxs]
         self.assertTrue(np.all(speeds_on_turns > 0.5 * max_target_speed), "Vehicle speed reduced more than 50% on turn")
 
-    def test_C1T_08_stops_close_to_destination(self):
+    def test_C1T_06_stops_close_to_destination(self):
```

Comment:
Didn't we talk about changing this metric?

Reply:
We discussed it but haven't ask for input from PMs or client